cdm/src/main/java/ucar/nc2/ft2/coverage/simpgeometry/SimpleGeometry.java

```diff
@@ -18,4 +18,6 @@
     public void setData(Array data);
     public Array getData();
 
+    public double[] getBBLower();
+    public double[] getBBUpper();

```

Comment:
Good call to add these methods to the interface! Good implementations of them too!